app/src/components/AddProposalPanel.js

```diff
@@ -14,23 +14,45 @@ const AddProposalPanel = ({ onSubmit }) => {
   }
   return (
     <form onSubmit={onFormSubmit}>
-      <Field label="Title">
+      <Field
+        label="Title"
+        css={`
+          margin: 16px 0;
+          div {
+            font-weight: 400;
```

Comment:
normal = 400!

Reply:
You are right, the `aragon/ui` Field component has a font-weight of 600, and in the figma it's 400. I think it's an inconsistency in the figma file, because the other apps do not change that weight on their panels. So I would prioritize the style defined in the `aragon/ui` components, at least in this case.